sdk/src/main/scala/com/akkaserverless/javasdk/impl/action/ActionHandler.scala

```diff
@@ -136,8 +136,6 @@ abstract class ActionHandler[A <: Action](protected val action: A) {
         throw new RuntimeException(
           s"No call handler found for call $name on ${action.getClass.getName}"
         )
-    } finally {
-      action._internalSetActionContext(Optional.empty())
```

Comment:
could be good with a comment here why we don't clear it

Reply:
True, I'll add. I started out with a boolean flag in the handler to detect and throw in case we'd forget and start invoking multiple commands on the same, but reverted that.